src/main/java/Duke.java

```diff
@@ -1,10 +1,151 @@
+// import libraries here
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    private static ArrayList<String> descriptions = new ArrayList<>();
+    private static ArrayList<Boolean> dones = new ArrayList<>();
+    private static ArrayList<String> types = new ArrayList<>();
+    private static ArrayList<String> dates = new ArrayList<>();
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        greeting();
+        getCommand();
     }
-}
+
+    private static void getCommand() {
+        String line;
+        Scanner in = new Scanner(System.in);
+        // get user input
+        line = in.nextLine();
+        while (!line.equals("bye")) {
+            if (line.equals("list")) {
+                showList();
+                line = in.nextLine();
+                continue;
+            }
+            if (line.startsWith("done")) {
+                int number = Integer.parseInt(line.substring(line.length() - 1));
+                markAsDone(number);
+                line = in.nextLine();
+                continue;
+            }
+            // Command is valid, handle the command
+            System.out.println("    ____________________________________________________________");
+            System.out.println("     Got it. I've added this task: ");
+            handleCommand(line);
+            System.out.println("     Now you have " + descriptions.size() + " tasks in the list.");
+            System.out.println("    ____________________________________________________________");
+            line = in.nextLine();
+        }
+        System.out.println("    ____________________________________________________________");
+        System.out.println("     Bye. Hope to see you again soon!");
+        System.out.println("    ____________________________________________________________");
+    }
+
+    private static void handleCommand(String line) {
```

Comment:
handleCommand method looks a little long, perhaps adding methods for adding different events can increase readability?